crio/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry.fedoraproject.org/fedora:25
+
+ENV VERSION=0 RELEASE=1 ARCH=x86_64
+LABEL com.redhat.component="cri-o" \
+      name="$FGC/cri-o" \
+      version="$VERSION" \
+      release="$RELEASE.$DISTTAG" \
+      architecture="$ARCH" \
+      usage="atomic install --system --system-package=no crio && systemctl start crio" \
+      summary="The cri-o daemon as a system container." \
+      maintainer="Yu Qi Zhang <jzehrarnyg@gmail.com>" \
+      atomic.type="system"
+
+RUN dnf -y --setopt=tsflags=nodocs install btrfs-progs-devel device-mapper-devel glib2-devel glibc-devel glibc-static gpgme-devel libassuan-devel libgpg-error-devel libseccomp-devel libselinux-devel \
```

Comment:
Yep, doing right now.

One main reason I wasn't very focused on this was that we were going to eventually use the packaged version, which will drop a lot of these packages. There are definitely more optimizable parts once we switch to that.